src/box/sql/build.c

```diff
@@ -2117,6 +2169,576 @@ sql_drop_table(struct Parse *parse_context)
 	sqlSrcListDelete(db, table_name_list);
 }
 
+/** Check is BOX constraint with given name exists. */
+static bool
+is_constraint_exists(struct tuple_constraint_def *cdef, uint32_t count,
```

Comment:
That is a weird english bit. Lets use `does_constraint_exist`. And make `cdef` const alongside.

Reply:
Dropped this function.